code/script.js

```diff
@@ -255,59 +270,165 @@ const checkQuestion = () => {
   // Compare the currentQuestion details with the secret person details in a different manner based on category (hair/eyes or accessories/others).
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
-  if (category === 'hair' || category === 'eyes') {
-
-  } else if (category === 'accessories' || category === 'other') {
-
+  if (category === 'hair') {
+      if (currentQuestion.value === secret.hair) {
+        keep = true
+        filterCharacters(keep)
+        console.log(keep)
+      } else {
+        keep  = false
+        filterCharacters(keep)
+        console.log(keep)
+      }
+  } else if (category === 'eyes') {
+    
+      if (currentQuestion.value === secret.eyes) {
+        keep = true
+        filterCharacters(keep)
+        console.log(keep)
+      } else {
+        keep  = false
+        filterCharacters(keep)
+        console.log(keep)
+      }
+  } else if (category === 'accessories') {
+    if (currentQuestion.value.includes('glasses')) {
+      if (currentQuestion.value.includes('glasses') === secret.accessories.includes('glasses')) {
+        keep = true
+        filterCharacters(keep)
+        console.log(keep)
+      } else if (currentQuestion.value.includes('glasses') !== secret.accessories.includes('glasses')) {
+        keep  = false
+        filterCharacters(keep)
+        console.log(keep)
+      }
+    } else if (currentQuestion.value.includes('hat')) {
```

Comment:
Is it possible to combine that accessories include both, hat and glasses? maybe like this
charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))